clip.md

```diff
@@ -0,0 +1,443 @@
+```k
+requires "kmcd-driver.md"
+requires "abaci.md"
+requires "dog.md"
+requires "spot.md"
+requires "vat.md"
+requires "vow.md"
+
+module PRE-CLIP
+    imports KMCD-DRIVER
+    imports ABACI
+    imports SPOT
+    imports VAT
+    imports VOW
+```
+
+Clip Configuration
+-----------------
+
+```k
+   syntax ClipStop ::= "noBreaker"
+                     | "noNewKick"
+                     | "noNewKickOrRedo"
+                     | "noNewKickOrRedoOrTake"
+
+   syntax Bool ::= ClipStop "<ClipStop" ClipStop [function]
+ // --------------------------------------------
```

Comment:
general nit: these separators don't match the length of the line above them in a lot of cases (Everett will complain about this, I know from experience)

Reply:
formatting errors fixed